### ISSUE TITLE
Love the book so far and a suggestion

### ISSUE THREAD

**Issue by matt-the-ogre** [brennen/userland-book#2]
Thanks so much. This is great so far. 

My suggestion is to consider a chapter / section on 'cron' for the book. I'm exploring it now and like setting up scripts / commands to run at various intervals.

Cheers, Matt

**Comment by brennen** [comment]
Yeah, I think a chapter on scheduling tasks would probably fit well.  I'll put it on the list. Thanks for the feedback!
